src/utils/ConfigLoader.ts

```diff
@@ -0,0 +1,12 @@
+import * as dotenv from 'dotenv';
+
+const DEFAULT_ENV_FILE = `${__dirname}/../../.env.local`;
+
+export const loadConfig = () => {
+  if (process.env.NODE_ENV) {
+    const path = `${__dirname}/../../.env.${process.env.NODE_ENV}`;
+    return dotenv.config({ path });
+  }
+  return dotenv.config({ path: DEFAULT_ENV_FILE });
```

Comment:
I suggest we do not load local file by default because it will load it in other non local environment.

As discussed I see this to he following way:
- load default `.env` file
- load `.env.[environment]` file if `NODE_ENV` is set

Beside above we should keep `.env.local` file for local environment but ask ourselves to do `NODE_ENV=local npm start` locally. Alternatively we could add it to start script in package.json (subject to future change when we figured out how to run it in cloud)